### PR TITLE
Update to babel v6 and others deps

### PR DESCRIPTION
**Pull request by ramasilveyra**
Update to `babel` v6 with `preset-es2015`, new api on `del` package, and the update to the last version of the others packages hasn't any breaking change.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -161,8 +161,7 @@ gulp.task('html', () => {
 });
 
 // Clean output directory
-gulp.task('clean', cb => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true},
-  cb));
+gulp.task('clean', () => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true}));
```
Comment (sindresorhus) [inline gulpfile.babel.js]: Unrelated to this PR, but thought I would mention that the `dot` option isn't necessary here as explicit dot files/folders are always included. - Author of `del`. That means you could probably remove `'!dist/.git'` as it shouldn't match dotfiles when using globstar matching (needs testing though just to be sure).

File: package.json [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+    "gulp-sourcemaps": "^1.6.0",
+    "gulp-uglify": "^1.4.2",
+    "gulp-uncss": "^1.0.4",
+    "gulp-useref": "^2.0.0",
+    "jshint-stylish": "^2.0.1",
+    "psi": "^1.0.6",
     "require-dir": "^0.3.0",
-    "run-sequence": "^1.0.1",
+    "run-sequence": "^1.1.4",
     "sw-precache": "^2.2.0"
```
Comment (sindresorhus) [inline package.json]: Many of these versions do not need to be bumped.

https://nodesource.com/blog/semver-tilde-and-caret

Comment (ramasilveyra) [inline package.json]: Thanks, I didn't know that. I will restore back all unnecessary version increments.

**Comment by ramasilveyra** [comment]
I will close this PR because I made a huge mistake... I commit on master.

File: gulpfile.babel.js (hunk 1 above)
Comment (ramasilveyra) [inline gulpfile.babel.js]: Just to help the future PR that fix that, this is the PR who added `, {dot: true}:` #537

**Pull request closed without merging**